lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -92,6 +92,11 @@ public static function get_theme_data( $deprecated = array(), $settings = array(
 
 			// Classic themes without a theme.json don't support global duotone.
 			$theme_support_data['settings']['color']['defaultDuotone'] = false;
+
+			// Allow themes to enable appearance tools via theme_support
```

Comment:
I think there should be a full-stop at the end of this comment, from reading the unit test feedback.